src/main/java/org/aist/aide/mappingservice/domain/models/Services.java

```diff
@@ -0,0 +1,12 @@
+package org.aist.aide.mappingservice.domain.models;
+
+public enum Services {
+    GENERIC,
```

Comment:
Just a note, we need a centralized way to manage these in the future.

Reply:
I was thinking this would either go back to a collection in mongo, or to whatever library we put some common stuff on.